### DISCUSSION TITLE
Does Budibase support dynamic sorting?

### DISCUSSION THREAD

**Discussion by thanhpd56** [Budibase/budibase#15007] [Help]
Hello! Does Budibase support dynamic sorting with DataProvider?
For example, when users click dropdown to select column to sort, then data will change according.
Thanks.

**Comment by ConorWebb96** [comment]
Hey @thanhpd56,

This is likely what you are after. https://docs.budibase.com/docs/searchfilter-data#dynamic-filtering I hope it helps.
